template_snippets/cloudconfig/g8s_master.yaml

```diff
@@ -194,6 +194,31 @@
       --http=127.0.0.1:4040
       ExecStop=-/usr/bin/docker stop -t 10 $NAME
       ExecStopPost=-/usr/bin/docker rm -f $NAME
+  - name: g8s-master-api-ping.service
+    enable: true
+    command: start
+    content: |
+      [Unit]
+      Description=g8s-master-api-ping Service
+      Requires=k8s-addons.service
```

Comment:
This should require the api server. So that when the api server unit is stopped ping is stopped too (with the healthcheck)